templates/debug/scheduled-crons.php

```diff
@@ -73,7 +84,18 @@
 
 	$sec_text = fs_text_x_inline( 'sec', 'seconds' );
 ?>
-<h1><?php fs_esc_html_echo_inline( 'Scheduled Crons' ) ?></h1>
+<?php echo $title_tag_open; ?>
+<?php
+	if ($display_toggle_button) {
```

Comment:
@DanieleAlessandra How about this instead?

```
<?php if ( $display_toggle_button ) : ?>
<button class="toggle-table" aria-expanded="<?php echo $is_open ?>">
    <span class="caret-icon">▼</span>
</button>
<?php endif ?>
```

Also, in our convention, we don't add a semicolon for single-line PHP code.